src/config.c

```diff
@@ -2539,6 +2539,12 @@ int updateClusterFlags(const char **err) {
     return 1;
 }
 
+static int updateClusterPort(const char **err) {
+    UNUSED(err);
+    clusterUpdateMyselfPort();
```

Comment:
Can we also add a simple test here, perhaps in `tests/unit/cluster/announced-ports` to validate the code change? You can look at other tests in `tests/unit/cluster` for some examples that should be easy to follow.

Reply:
@DevineLiu Any update on writing a test for this?